framework/sac_rlhf.py

```diff
@@ -447,7 +447,9 @@ def get_action(self, x):
         for idx, trunc in enumerate(truncations):
             if trunc:
                 real_next_obs[idx] = infos["final_observation"][idx]
-        rb.add(obs, real_next_obs, actions, groundTruthRewards, terminations, infos)
+
+        rewards = reward_net.predict_reward(obs, actions)
```

Comment:
You need to use rewards = rewards.detach().cpu().numpy() here.

Reply:
fair :+1: